### ISSUE TITLE
All buttons should be relocated to a navigation bar on the top

### ISSUE THREAD

**Issue by VeenaArv** [BUGS-NYU/test_site#13]
It's confusing having the About and Mozilla Affiliation separate from the rest. A navigation bar on top will be much cleaner to look at in terms of design and more intuitive to navigate.

**Comment by S1MB10T3** [comment]
Personally feel that postion of the about and mozilla affilation links provides a better exprience for the user. A student visting the site would like to know:

1. What the club is about
2. Events/meeting times and locations
3. Current/pervious projects
4. Github link

The homepages allows for students who already are in the club or seek to join the club as easy and simple navigation to get what they need.

**Comment by KEYS248** [comment]
I agree with Rokney and also moved the Connect page with About and Mozilla, for the reason of what new people vs old members would be looking for. I'll leave this issue open tho because it may be nice to see what the website would look like with a nav bar as opposed to a drop down menu, but it is non-essential to current functionality
